frontend/package.json

```diff
@@ -58,10 +60,10 @@
     "optimize-css-assets-webpack-plugin": "^5.0.4",
     "popper.js": "^1.14.3",
     "prop-types": "^15.7.2",
-    "react": "^16.14.0",
```

Comment:
Did react need to be downgraded? Also we should probably keep the `^` if at all possible. That's supposed to ensure small bug fix updates get incorporated.

Reply:
No, it needed to be upgraded from `16.14` to `16.8` (for now, but we'll probably further upgrade on the IR&D development.)

Reply:
I think 16.8 is earlier though? https://legacy.reactjs.org/versions/